docs/cicd/others-script-cd-template.sh

```diff
@@ -0,0 +1,52 @@
+# This is just an example workflow for continous deployment.
+# You should customize it to meet your own requirements.
+# Also you should export the following environment variables for Azure/M365 login:
+# AZURE_ACCOUNT_NAME
+# AZURE_ACCOUNT_PASSWORD
+# AZURE_SUBSCRIPTION_ID
+# AZURE_TENANT_ID
+# M365_ACCOUNT_NAME
+# M365_ACCOUNT_PASSWORD
+
+# If the hosting environment is not provisioned, set this environment variable to false.
+# or if it's provisioned and has not updates, set this environment variable to true.     
+# RUN_PROVISION
+
+
+# Setup environment.
+sudo apt install -y nodejs npm curl git
+sudo npm install -g @microsoft/teamsfx-cli 
```

Comment:
why do we need sudo?

Reply:
also shouldn't we use a local version of teamsfx and not a global one? perhaps aliased through npm scripts so it's easy to run? This way you could ensure you version lock your project to a specific teamsfx version

Reply:
> why do we need sudo?

Usually, when installing packages for the system, high level privileges will be required, adding `sudo` is a way for this purpose.

Reply:
> also shouldn't we use a local version of teamsfx and not a global one? perhaps aliased through npm scripts so it's easy to run? This way you could ensure you version lock your project to a specific teamsfx version

The idea of using local version of teamsfx-cli is better, will update it later.

Reply:
if high level privileges are required for installation of stuff like this, something is misconfigured. Just because we think something is "usual" doesn't mean we should make it a standard practice. Please take a look at how this can be done without sudo. Using sudo is not a best practice.

Reply:
I'm okay to remove `sudo`, but will add comments to remind users to prepare efficient permissions.

Reply:
`sudo` removed, and comments to remind users are added.

Reply:
@qinezh FYI